test/testpages/dragdrop.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<link rel="shortcut icon" href="../resources/images/favicon.ico" type="image/x-icon">
+		<title>Dragon Drop testing</title>
+		<script>
+			function allowDrop(ev, targetElement) {
+				ev.preventDefault();
+				if(targetElement){
+					targetElement.style.backgroundColor="DarkGreen";
```

Comment:
could the couple of styling oprations be single sourced into the same css class? then you could just call classlist.add

Reply:
Sure.